### PR TITLE
Various fixes for visualization.

### PR DESCRIPTION
**Pull request by ghost**
Many minor fixes, like added missing urls to asset, cleaned irrelevant
fields.

### CODE REVIEW

File: src/ralph/dc_view/serializers/models_serializer.py [hunk 1]
```diff
@@ -43,8 +43,19 @@ def admin_link(self, obj):
 
 class DataCenterAssetSerializerBase(serializers.ModelSerializer):
     model = serializers.CharField(source='model.name')
-    service = serializers.CharField(source='service.name')
-    orientation = serializers.CharField(source='model.get_orientation_desc')
+    service = serializers.SerializerMethodField('get_service_env')
+    orientation = serializers.CharField(source='get_orientation_desc')
+    url = serializers.CharField(source='get_absolute_url')
+
+    def get_service_env(self, obj):
```
Comment (mkurek) [inline src/ralph/dc_view/serializers/models_serializer.py:49]: what about environment?

Comment (ghost) [inline src/ralph/dc_view/serializers/models_serializer.py:49]: It's absent in current Ralph.

**Comment by mkurek** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**